justfile

```diff
@@ -59,7 +60,7 @@ install-jobrunner: check install-opensafely-user
 install-release-hatch: check
   ./services/release-hatch/install.sh
 
-install-airlock: check
+install-airlock: check install-opensafely-user
```

Comment:
there's a trade-off here - on the downside this will mean that `install-opensafely-user` gets run twice when we run `just manage`, but on the upside this will mean that if we update the airlock config & run `just install-airlock` the updated config will be correctly pulled in.

(if we were using `chef`, it would de-duplicate the list of targets to run & only run this once, but `just` has a different conceptual model)

Reply:
yeah, as long as install-opensfaely-user is idempotent, I think this trade off is worth it